server/src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -71,10 +71,10 @@ Environment environment() {
         return nodeEnv.get();
     }
 
-    void exitWithUserException(int exitCode, Exception e) {
-        err.print(USER_EXCEPTION_MARKER);
```

Comment:
The stderr pump thread looks for this marker, I wonder if this changes that behaviour?

Reply:
It slightly changes the error path, but it effectively doesn't change the result because the pump still exits after the stream is closed, which happens when we exit the main process.

I plan to create a followup to remove USER_EXCEPTION_MARKER as it is now no longer used.

Reply:
Oh, I see now, the error pump finally has the latch decrement.